### ISSUE TITLE
Improvement: Update internal `torch.norm` calls to `torch.linalg.norm`

### ISSUE THREAD

**Issue by KarahanS** [qdrant/quaterion#203]
In various places, we are using `torch.norm` to calculate the vector norms of the inputs (for example [here](https://github.com/qdrant/quaterion/pull/202/files#diff-cf863b8afd776fc89303b377aeb9f57d425409a9b6e6f8ce07faeab2008bc71d)) . According to the documentation:
> ```torch.norm``` is deprecated and may be removed in a future PyTorch release. Its documentation and behavior may be incorrect, and it is no longer actively maintained.

It's better to switch to alternatives such as [torch.linalg.norm](https://pytorch.org/docs/stable/generated/torch.linalg.norm.html#torch.linalg.norm) or [torch.nn.function.normalize](https://pytorch.org/docs/stable/generated/torch.nn.functional.normalize.html) if applicable.

**Comment by KarahanS** [comment]
I'll be working on this after the PR #202 is merged/closed.

**Comment by helios2003** [comment]
Hello, this is Ankit..... Can I work on this issue?

**Comment by helios2003** [comment]
Hello this is Ankit ....and I like to contribute open source. I found this particular issue interesting. Can I be assigned this issue?
